posts/zzz_DO_NOT_EDIT_document_yo.../document_your_universe_with_dverse.qmd

```diff
@@ -0,0 +1,186 @@
+---
+title: "Creating a better universe with {dverse}"
```

Comment:
Just curious, what does the "d" stand for in "dverse"?

Reply:
Ah probably `document universe`, aye?

Reply:
Exactly! Happy to add it -- and also very happy to see that although it might not be obvious it's somewhat intuitive. I think the clearest hint is the call `dverse::document_universe()`.